client/state/plugins/installed/actions.js

```diff
@@ -191,7 +191,7 @@ function installPluginHelper( siteId, plugin, isMainNetworkSite = false ) {
 		dispatch( { ...defaultAction, type: PLUGIN_INSTALL_REQUEST } );
 
 		const doInstall = function( pluginData ) {
-			return getPluginHandler( siteId, pluginData.id ).install();
+			return getPluginHandler( siteId, pluginData.slug ).install();
```

Comment:
Ahh okay - When I looked earlier I thought it was, but it looks like all those installPlugin calls were for https://github.com/Automattic/wp-calypso/blob/55cfc41642f8d444493893208a483d391850c2e1/client/state/plugins/premium/actions.js#L309.